### ISSUE TITLE
Active Directory authentication fails after upgrade from 1.7.0 to 1.7.1

### ISSUE THREAD

**Issue by sorin-p** [go-gitea/gitea#5921]
- Gitea version (or commit ref): 1.7.1
- Git version: 2.11.0
- Operating system:
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [X] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [X] Not relevant
- Log gist:
```
2019/02/01 10:18:25 [T] Dialing LDAP with security protocol (1) without verifying: true
2019/02/01 10:18:26 [T] LDAP will bind directly via UserDN template: CN=%s,CN=Users,DC=blabladomain,DC=com
2019/02/01 10:18:26 [T] Binding with userDN: CN=UserName,CN=Users,DC=blabladomain,DC=com
2019/02/01 10:18:26 [D] LDAP auth. failed for CN=UserName,CN=Users,DC=blabladomain,DC=com, reason: LDAP Result Code 200 "Network Error": ldap: response channel closed
2019/02/01 10:18:26 [D] Template: user/auth/signin
2019/02/01 10:18:26 [I] Failed authentication attempt for UserName from ...

```
## Description
Logging in with users from Active Directory source does not work after upgrade to v1.7.1
Deleted, recreated the users and the same error persists. Some fragments from the log were replaced with dummy values.

**Comment by lafriks** [comment]
What OS are you run gitea on?

**Comment by sorin-p** [comment]
Linux armv7l

**Comment by nigeldixon** [comment]
Same report, but different error:
`ldap: recovered panic in processMessages: runtime error: invalid memory address or nil pointer dereference`

I'm running in docker but also on armv7 with SQLite db

**Comment by mootboy** [comment]
Is this fixed by #6107?

**Comment by lafriks** [comment]
I think it should be, please reopen if it is not
